### PR TITLE
Refactor/BouncyUpwardCheck

### PR DESCRIPTION
**Pull request by m4rtint**

### CODE REVIEW

**Comment by m4rtint** [comment]
[Upward Check for bouncy (0.5)](https://trello.com/c/PXNUPY8S/912-upward-check-for-bouncy-05)

**Review by Gursaanj — APPROVED** [review]

File: Assets/_PartyParkade/Script/PowerUp/BouncyObject.cs [hunk 1]
```diff
@@ -65,6 +65,18 @@ protected override void SetDeactivateScalingAnimation()
 
         protected override bool CanThrowBackwards(PlayerController playerController)
         {
+            Vector3 positionAboveCar = GetPositionToRayCastBackwards(playerController.Car);
+            Vector3 direction = playerController.Car.transform.TransformDirection(Vector3.up);
+            RaycastHit hit;
+            if (Physics.Raycast(positionAboveCar, direction, out hit, UpwardsCheckRaycastDistance))
+            {
+                Debug.DrawRay(positionAboveCar, direction * UpwardsCheckRaycastDistance, Color.red, 10f);
+                Debug.Log("Cannot throw backwards, hit :" + hit.transform.name);
```
Comment (wbeltran777) [inline Assets/_PartyParkade/Script/PowerUp/BouncyObject.cs:74]: Should we remove these debugs or will we address all the leftover debugs in our project on a different pass?

Comment (m4rtint) [reply Assets/_PartyParkade/Script/PowerUp/BouncyObject.cs:74]: Address it in the end, since deployable is still not perfect yet.

**Review by wbeltran777 — APPROVED** [review]

**Comment by m4rtint** [comment]
[Deployable object bugs out when deployed under a bridge](https://trello.com/c/YwWOnR2u/910-deployable-object-bugs-out-when-deployed-under-a-bridge)

**Pull request merged**